lms/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,119 @@
+// LMS -- utilities -- mixins
```

Comment:
TODO: I'll talk to someone who's a Git ninja about why this file was deleted/re-created vs. moved in the Git history.